docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -64,8 +63,161 @@ For more information, refer to
+(Required, string) The JSON key in get case response that contains the external
+case title.
+
+`getIncidentUrl`::
+(Required, string) The REST API URL to get the case by ID from the third-party
+system. If you are using the `xpack.actions.allowedHosts` setting, make sure the
+hostname is added to the allowed hosts. You can use a variable to add the
+external system ID to the URL. For example:
++
+[source,text]
+----
+https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}
+----
++
+NOTE: Due to Mustache template variables (the text enclosed in triple braces,
+for example, `{{{case.title}}}`), the JSON is not validated when you create the
+connector. The JSON is validated after the Mustache variables have been placed
+when REST method runs. Manually ensure that the JSON is valid, disregarding the
+Mustache variables, so the later validation will pass.
+
+`hasAuth`::
+(TBD, boolean) If true, a username and password for login type authentication
+must be provided.
+
+`headers`::
+(Optional, string) A set of key-value pairs sent as headers with the request
+URLs for the create case, update case, get case, and create comment methods.
+
+`updateIncidentJson`::
+(Required, string) The JSON payload sent to the update case URL to update the
+case. You can use variables to add Kibana Cases data to the payload. Required
+variables are `case.title` and `case.description`. For example:
++
+[source,json]
+----
+{
+	"fields": {
+	  "summary": {{{case.title}}},
+	  "description": {{{case.description}}},
+	  "labels": {{{case.tags}}}
+	}
+}
+----
++
+NOTE: Due to Mustache template variables (which is the text enclosed in triple
+braces, for example, `{{{case.title}}}`), the JSON is not validated when you
+create the connector. The JSON is validated after the Mustache variables have
+been placed when REST method runs. Manually ensure that the JSON is valid to
+avoid future validation errors; disregard Mustache variables during your review.
+
+`updateIncidentMethod`::
+(Optional, string) The REST API HTTP request method to update the case in the
+third-party system. Valid values are `patch`, `post`, and `put`. The default
+value is `put`.
+
+`updateIncidentUrl`::
+(Required, string) The REST API URL to update the case by ID in the third-party
+system. You can use a variable to add the external system ID to the URL. If you
+are using the `xpack.actions.allowedHosts` setting, make sure the hostname is
+added to the allowed hosts. For example:
++
+[source,text]
+----
+https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.ID}}}
+----
+     
+`viewIncidentUrl`::
+(Required, string) The URL to view the case in the external system. You can use
+variables to add the external system ID or external system title to the URL.For example:
++
+[source,text]
+----
+https://testing-jira.atlassian.net/browse/{{{external.system.title}}}
+----
+
+//For more information, refer to
+//<<cases-webhook-action-type>>.
```

Comment:
> //<<cases-webhook-action-type>>.

This link cannot be used until https://github.com/elastic/kibana/pull/137726 is merged.